src/main/java/org/triplea/ui/TripleaMainMenu.java

```diff
@@ -0,0 +1,175 @@
+package org.triplea.ui;
+
+import java.awt.Desktop;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.text.MessageFormat;
+
+import games.strategy.debug.ClientLogger;
+import games.strategy.engine.ClientContext;
+import games.strategy.triplea.UrlConstants;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Button;
+import javafx.scene.control.PasswordField;
+import javafx.scene.control.TextField;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+
+public class TripleaMainMenu extends BorderPane {
+
+  private final TripleA triplea;
+  private final TripleaDownload downloadPane;
+  private final TripleaSettings settingsPane;
+
+  @FXML
+  private Text loggedInText;
+
+  @FXML
+  private HBox loginForm;
+
+  @FXML
+  private TextField username;
+
+  @FXML
+  private PasswordField password;
+
+  @FXML
+  private Button buttonBack;
+
+  @FXML
+  private Text version;
+
+  @FXML
+  private VBox aboutSection;
+
+  @FXML
+  private HBox gameOptions;
+
+  @FXML
+  private VBox mainOptions;
+
+  /**
+   * Public constructor.
+   * @param triplea The root pane
+   * @throws IOException If the FXML file is not present
```

Comment:
Just FYI for future reference...  You can always add a period to the end of the `@param` (or whatever tag is first) description to make Checkstyle happy. :smile:  It's not necessary to add a summary.